File: src/main/java/com/github/caijh/commons/util/process/ProcessResult.java
package com.github.caijh.commons.util.process;


/**
 * 调用命令的执行结果.
 */
public class ProcessResult {

    private int exitValue = -1;

    private String consoleString;

    public boolean isSuccessful() {
        return exitValue == 0;
    }

    public int getExitValue() {
        return exitValue;
    }

    public void setExitValue(int exitValue) {
        this.exitValue = exitValue;
    }

    public String getConsoleString() {
        return consoleString;
    }

    public void setConsoleString(String consoleString) {
        this.consoleString = consoleString;
    }

}

File: src/main/kotlin/com/github/caijh/commons/util/process/ProcessResult.kt
package com.github.caijh.commons.util.process

/**
 * 调用命令的执行结果.
 */
class ProcessResult {
    var exitValue = -1
    var consoleString: String? = null
    val isSuccessful: Boolean
        get() = exitValue == 0
}
